TooltipDecorator/tests/TooltipDecorator-specs.js

```diff
@@ -27,111 +21,51 @@ describe('TooltipDecorator Specs', () => {
 		const tooltipText = 'This is a tooltip';
-
-		const subject = mount(
+		render(
 			<Root>
-				<FloatingLayerBase>
+				<FloatingLayerBase data-testid="floatingLayer">
 					<Tooltip>
 						{tooltipText}
 					</Tooltip>
 				</FloatingLayerBase>
 			</Root>
 		);
 
-		const expected = '';
-		const actual = subject.find('FloatingLayer').text();
+		const actual = screen.queryByText(tooltipText);
 
-		expect(actual).toBe(expected);
+		expect(actual).toBeNull();
 	});
 
-	test('should apply alignment when `centered` and `marquee`', () => {
-		const subject = shallow(
+	test('should apply `text-align: center` style when `centered=true` and marquee=true', () => {
+		render(
 			<TooltipLabel centered marquee>
 				Label
 			</TooltipLabel>
 		);
 
 		const expected = 'center';
-		const actual = subject.prop('alignment');
+		const actual = screen.getByText('Label');
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveStyle({'text-align': expected});
 	});
 
-	test('should not apply alignment when `centered` but not `marquee`', () => {
-		const subject = shallow(
-			<TooltipLabel centered>
+	test('should not apply `text-align: center` style when `centered=false`', () => {
+		render(
+			<TooltipLabel centered={false}>
 				Label
 			</TooltipLabel>
 		);
 
-		expect(subject).not.toHaveProperty('alignment');
-	});
-
-	test('should position Tooltip above when tooltipPosition equals "above"', () => {
-		const tooltipText = 'This is a tooltip';
-
-		const subject = mount(
-			<TooltipButton tooltipPosition="above">
-				{tooltipText}
-			</TooltipButton>
-		);
-
-		const expected = 'above';
-		const actual = subject.find('TooltipDecorator').first().prop('tooltipPosition');
-
-		expect(actual).toBe(expected);
-	});
-
-	test('should position Tooltip below when tooltipPosition equals "below"', () => {
-		const tooltipText = 'This is a tooltip';
-
-		const subject = mount(
-			<TooltipButton tooltipPosition="below">
-				{tooltipText}
-			</TooltipButton>
-		);
-
-		const expected = 'below';
-		const actual = subject.find('TooltipDecorator').first().prop('tooltipPosition');
-
-		expect(actual).toBe(expected);
-	});
-
-	test('should position Tooltip on left when tooltipPosition equals "left middle"', () => {
-		const tooltipText = 'This is a tooltip';
-
-		const subject = mount(
-			<TooltipButton tooltipPosition="left middle">
-				{tooltipText}
-			</TooltipButton>
-		);
-
-		const expected = 'left middle';
-		const actual = subject.find('TooltipDecorator').first().prop('tooltipPosition');
-
-		expect(actual).toBe(expected);
-	});
-
-	test('should position Tooltip on right when tooltipPosition equals "right middle"', () => {
-		const tooltipText = 'This is a tooltip';
-
-		const subject = mount(
-			<TooltipButton tooltipPosition="right middle">
-				{tooltipText}
-			</TooltipButton>
-		);
-
-		const expected = 'right middle';
-		const actual = subject.find('TooltipDecorator').first().prop('tooltipPosition');
```

Comment:
Note to reviewer: Last 4 tests were deleted because the `tooltipPosition` prop is not visible in DOM